subworkflows/local/snv_annotation/main.nf

```diff
@@ -22,12 +27,29 @@ workflow SNV_ANNOTATION {
     BCFTOOLS_FILLTAGS_ANNO(ECHTVAR_ANNO.out.bcf)
     ch_versions = ch_versions.mix(BCFTOOLS_FILLTAGS_ANNO.out.versions)
 
-    BCFTOOLS_FILLTAGS_ANNO.out.vcf
-        .map{ meta, vcf -> [ meta, vcf, [] ] }
-        .set { vep_in }
+    // Annotating with CADD
+    if (val_annotate_cadd) {
```

Comment:
Is it possible to check if the channel `ch_cadd_resources` is empty or an empty array here instead of checking the params.cadd_resources?

Reply:
Would that really be better? I don't think there's a way to do that with an if statement, because wouldn't you need to check if the content of the channel is an empty array with branch/filter or something then?

Reply:
Yeah you're right you would probably need to "unpack" the channel with a map statement. Let's leave it as is